core/src/main/java/org/apache/xtable/delta/DeltaConversionTarget.java

```diff
@@ -268,9 +281,23 @@ private void commitTransaction() {
     }
 
     private Map<String, String> getConfigurationsForDeltaSync() {
+
+      // The output of detail() method has only one row with the following schema.
+      // deltaTable.detail() is added to the constructor for this class, and sets 
+      // a private variable to of deltaTable 
+
+      // limit the results to the attributes needed
+      Dataset<Row> record = deltaTable.detail().select("minWriterVersion", "minReaderVersion");
```

Comment:
Can you get this information from `deltaLog.snapshot().metadata()`?

Reply:
I dont think so.  I looked through the api for the `deltaLog.snapshot().metadata()`, and did not find a referance to it.  The only place i found it at was here:
[](https://docs.delta.io/latest/delta-utility.html)

Reply:
It would be good to see if we can just let the Delta Lake library automatically set the versions for us. Can you try that as well? If I remember correctly, it looked like it would auto update them for us.